prow/plugins/plugins.go

```diff
@@ -712,6 +808,15 @@ func validateConfigUpdater(updater *ConfigUpdater) error {
 	return nil
 }
 
+func validateRequireMatchingLabel(rs []RequireMatchingLabel) error {
+	for i, r := range rs {
+		if err := r.validate(); err != nil {
+			return fmt.Errorf("error validating the %dth require_matching_label config: %v", i, err)
```

Comment:
error validating the 0th...
    error validating the 1th...

/shrug

Reply:
I think the user can figure out what it means haha. This is just an error message so proper grammar isn't worth the extra code in my opinion.

Reply:
sure